wfe/wfe.go

```diff
@@ -1627,6 +1638,62 @@ func (wfe *WebFrontEndImpl) makeChallenges(authz *core.Authorization, request *h
 	return nil
 }
 
+// validateReplacementOrder performs several sanity checks on the order to
+// determine if the order is a replacement of an existing order. If the order is
+// not a replacement or is a valid replacement, no problem will be returned.
+// Otherwise the caller will receive a problem.
+func (wfe *WebFrontEndImpl) validateReplacementOrder(newOrder *core.Order) *acme.ProblemDetails {
+	if newOrder == nil {
+		return acme.InternalErrorProblem("Order is nil")
+	}
+
+	// Lock the order for reading
+	newOrder.RLock()
+	defer newOrder.RUnlock()
+
+	if newOrder.Replaces == "" {
+		wfe.log.Printf("Order %q is not a replacement\n", newOrder.ID)
```

Comment:
Probably remove this debug print.

Reply:
I wanted this and other `wfe.log.Print` lines here on purpose.